selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/intelligencecommand/MacrosCommandsEditorTest.java

```diff
@@ -142,7 +141,6 @@ public void checkCommandMacrosIntoPreviewUrl() {
     commandsEditor.selectMacroCommand("${server.wsagent/http}");
     commandsEditor.typeTextIntoEditor(Keys.ARROW_UP.toString());
     commandsEditor.typeTextIntoEditor(Keys.SPACE.toString());
-    commandsEditor.waitMacroCommandIsSelected("${server.wsagent/http}");
```

Comment:
Why did you decide to remove this verification?

Reply:
It hard to check a selection now